packages/component-library/src/components/plmg-slider/test/plmg-slider.e2e.ts

```diff
@@ -3,33 +3,149 @@ import { AxePuppeteer } from '@axe-core/puppeteer';
 import { Page } from 'puppeteer';
 
 describe('plmg-slider', () => {
-  it('renders', async () => {
+  it('renders the component', async () => {
     const page = await newE2EPage();
     await page.setContent(
-      '<plmg-slider name="Range Slider" range-values"0, 10"></plmg-slider>'
+      '<plmg-slider name="Range Slider" range-values="0, 10"></plmg-slider>'
     );
-
     const element = await page.find('plmg-slider');
-
     expect(element).toHaveClass('hydrated');
   });
-
-  it('default slider is accessible', async () => {
+  it('renders the input range', async () => {
     const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider >>> input[type="range"]');
+    expect(element).not.toBe(null);
+  });
+});
 
-    const htmlContent = `<plmg-slider name='Range Slider' range='0, 100'></plmg-slider>`;
+describe('sets the range and initial valuess', () => {
+  it('sets min and max values', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0.1, 100.31"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider >>> input[type="range"]');
+    expect(element).toEqualAttribute('min', '0.1');
+    expect(element).toEqualAttribute('max', '100.31');
+  });
+  it('sets the value to min if not default-value is set', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '0');
+  });
+  it('sets the inital value if default value is within range', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10" default-value="5"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '5');
+  });
+  it('sets the value to the min if default value prop is above the allowed range', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0, 10" default-value="10.001"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '0');
+  });
+  it('set the value to the min if the default value prop is below the allowed range', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<plmg-slider name="Range Slider" range-values="0.1, 9.99" default-value="-0.1"></plmg-slider>'
+    );
+    const element = await page.find('plmg-slider');
+    expect(element).toEqualAttribute('value', '0.1');
+  });
+});
 
-    await page.setContent('<main>' + htmlContent + '</main>');
+describe('displays marks', () => {
```

Comment:
I suggest adding "plmg-slider" to your test cases to be explicit they relate to this component.
```suggestion
describe('plmg-slider displays marks', () => {
```